### ISSUE TITLE
setOverlapAllowed in Entry class doesn't work

### ISSUE THREAD

**Issue by JaakkoHintsala** [stefanuebe/vaadin-fullcalendar#186]
public void setOverlapAllowed(boolean overlap) {
        setOverlap(true);
    }

The method ignores the overlap boolean parameter.

**Comment by stefanuebe** [comment]
Yep, the parameter is unnecessary. The method itself works as intended, I just forgot to remove the parameter.

**Comment by stefanuebe** [comment]
Fix released with [6.3.1](https://github.com/stefanuebe/vaadin-fullcalendar/releases/tag/6.3.1)
